dev-client/src/util.ts

```diff
@@ -24,20 +25,15 @@ import {
 } from 'terraso-client-shared/utils';
 
 import {COORDINATE_PRECISION} from 'terraso-mobile-client/constants';
+import {fallbackLocale} from 'terraso-mobile-client/translations';
 
 export const getSystemLocale = () => {
```

Comment:
Suggestion: Could the locale and the language logic live next to each other? Since they look very similar and are about similar things

Reply:
I created `localization.ts` for related functions and constants.